### PR TITLE
Add a setting to specify a folder where newly created journal notes will be placed.

### PR DESCRIPTION
**Pull request by sanderboom**
Refs pbek/QOwnNotes#795

### CODE REVIEW

**Comment by pbek** [comment]
please don't set `"minAppVersion": "17.12.0",`, that version isn't needed and not released yet

File: journal-entry/info.json [hunk 1]
```diff
@@ -2,8 +2,8 @@
   "name": "Journal entry",
   "identifier": "journal-entry",
   "script": "journal-entry.qml",
-  "version": "1.0.0",
-  "minAppVersion": "17.05.6",
+  "version": "1.1.0",
+  "minAppVersion": "17.12.0",
```
Comment (pbek) [inline journal-entry/info.json]: please don't set `"minAppVersion": "17.12.0",`, that version isn't needed and not released yet

Comment (sanderboom) [reply journal-entry/info.json]: Hmm ok. I did that b/c otherwise people will not be able to read the description.

Comment (pbek) [reply journal-entry/info.json]: I guess that's just an issue in QON, but doesn't break the script itself. :)

**Pull request merged**